alf/bin/train_play_test.py

```diff
@@ -165,6 +165,11 @@ def _to_conf_params(parameters):
 ]
 MBRL_TRAIN_PARAMS = _to_conf_params(MBRL_TRAIN_CONF)
 
+OAC_TRAIN_CONF = OFF_POLICY_TRAIN_CONF + [
+    'TrainerConfig.unroll_with_grad=True',
```

Comment:
Why do you need unroll_with_grad? I think OAC only changes the exploration but not the training.

Reply:
This was previously used for enabling grad computation during unroll. But already updated following Wei's reviews by using "with torch.enable_grad()" instead. Forgot to updated here.